### ISSUE TITLE
Question: What about this library makes it a React lib?

### ISSUE THREAD

**Issue by mpalmr** [foursquare/react-foursquare#7]
I don't see anything specific to React in this library. It looks like just a "JS SDK". If you provided a React component that abstracted the usage of it I would understand why this is a "React SDK". Just mentioning this as I found it confusing when looking through Squarespace libs and just wanted a general JS SDK for my frontend that isn't using React and it works perfectly fine.

Also as an aside are you open to contributions via PRs?

**Comment by christineywang** [comment]
@mpalmr, yes, it does look like you can submit PRs for review. If it's a major change or an idea you want to float around with the Foursquare team, it may be better to create an issue first so we can discuss the best approach to tackle the idea before code gets written. Thanks in advance for contributing!
